src/scss/grommet-core/_objects.image.scss

```diff
@@ -33,4 +35,14 @@
   &--full-vertical {
     height: 100%;
   }
+
+  &__image {
+    width: 100%;
+  }
+
+  &__caption {
+    display: flex;
```

Comment:
I think text-align: center is sufficient. We don't need flexbox for just a line of text.